### ISSUE TITLE
Switching between filters

### ISSUE THREAD

**Issue by kmorengithub** [scantailor/scantailor#251]
It seems to be smart to switch between filters with keys 1 to 6 (like 'next/previous' with 'w/q').  Sometimes a page needs fixing into filter 3 (geometric distortions) when I'm on the filter 5 or 6.  So, I want to switch to filter 3, correct and switch back (or to 4, 5).  Now I need to:
* move the cursor at the field of filters;
* change filter,
* and move cursor back at the field of view to correct result.
I think this hotkeys will be very helpful.

**Comment by atla5** [comment]
Just to clarify, by 'filter' you mean these, right?:
![image](https://cloud.githubusercontent.com/assets/5565284/19501653/39cf93e8-9575-11e6-8f41-f9d0b887be61.png)

**Comment by kmorengithub** [comment]
Yes, because `scantailor-cli` have options `--start-filter` and `--end-filter`. That's the same, isn't it?
